server/datastore/mysql/migrations/tables/20240424124712_AddSoftwareInstallerTables.go

```diff
@@ -120,7 +120,7 @@ CREATE TABLE IF NOT EXISTS host_software_installs (
     REFERENCES software_installers (id)
     ON DELETE CASCADE ON UPDATE CASCADE,
 
-  UNIQUE KEY idx_host_software_installs_host_installer (host_id, software_installer_id),
+  KEY idx_host_software_installs_host_installer (host_id, software_installer_id),
```

Comment:
Heads-up folks! You will have to `make db-reset` to have the updated migration, assuming you've already run the old one (@ghernandez345 @roperzh @gillespi314 @dantecatalfamo ).